common/data/microcopy.tsx

```diff
@@ -162,7 +162,7 @@ export const pastExhibitionsStrapline =
 // This is the label used in the search box, both for the
 // search in the global nav and on the categories in wc.org/search
 export const searchLabelText = {
-  overview: 'Search our stories, images, catalogue and events',
+  overview: 'Search for anything',
```

Comment:
This will show even for those without the toggle, is that ok?

Reply:
good point. Probably not. Will fix